### PR TITLE
Blog post about new pre workshop help session

### PR DESCRIPTION
**Pull request by rgaiacs**
@k8hertweck and @raynamharris Could you review and let me know when ready to merge?

### CODE REVIEW

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+We invite everyone committed with one upcoming workshop, independently of the
+organization stage (before decide dates, preparing workshop homepage,
+registrations in progress, reviewing lessons that will be delivery),
+to join the helping session, sharing experience, have a good discussion
+and meet others instructors.
+</p>
+<p>
+If you can't attend this first helping session, don't worry.
+There will be a second helping session on the begin of 2016,
+date need to be defined,
```
Comment (rgaiacs) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: @k8hertweck Do we have a date?

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+author: Raniere Silva
+title: "Launching Pre Workshop Helping Session"
+date: 2015-12-04
+time: "10:00:00"
+category: ["Mentoring", "Community"]
+---
+<!-- start excerpt -->
+<p>
+Organizing one workshop isn't easy
+and sometimes things out of your control when wrong in the last minute
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: “when“ → “go”.

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+like the internet access on the room is very slow.
+After months of discussions of what could be done to help instructors
+and hosts when organizing their workshops and being prepared for the show time
+the Mentoring Subcommittee decided to test some helping sessions.
+</p>
+<!-- end excerpt -->
+<p>
+The first helping session will happen on
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: “helping session” → “help session” throughout the post.

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+<a href="http://pad.software-carpentry.org/pre-workshop-morning">December 9th at 3PM UTC</a>
+and <a href="http://pad.software-carpentry.org/pre-workshop-evening">December 10th at 12PM UTC</a>
+to accommodate different timezones.
+Instructions of how to connect can be found in the previous links.
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: “Instructions of how to connect” → “Connection instructions”?

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+</p>
+<p>
+We invite everyone committed with one upcoming workshop, independently of the
+organization stage (before decide dates, preparing workshop homepage,
+registrations in progress, reviewing lessons that will be delivery),
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: I'd just drop “that will be delivery”.

**Comment by k8hertweck** [comment]
Not sure on date yet; sometime in January? Perhaps we can talk as a subcommittee about whether we want these occurring on the same weeks as debriefings, or put them on alternate weeks?

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+to join the helping session, sharing experience, have a good discussion
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: “sharing experience” → “share experience”.

File: blog/2015/12/lauching-pre-workshop-help-session.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+and meet others instructors.
+</p>
+<p>
+If you can't attend this first helping session, don't worry.
+There will be a second helping session on the begin of 2016,
+date need to be defined,
+and it will probably become a regular event.
+More important, if you want a one-to-one chat about the workshop you are
+organizing or going to teach we are happy to do that
+and for that just send a email to <a href="mailto:mentoring@lists.software-carpentry.org">mentoring@lists.software-carpentry.org</a>.
```
Comment (wking) [inline blog/2015/12/lauching-pre-workshop-help-session.html]: “that and for that just send a email to” → “that; just email”

**Pull request merged**